configuration.go

```diff
@@ -173,6 +175,43 @@ type TLS struct {
 // Certificates defines traefik certificates type
 type Certificates []Certificate
 
+//CreateTLSConfig creates a TLS config from Certificate structures
```

Comment:
Could you explain in the comment that Certs could be either a file path, or the file content itself ?